src/core/components/headers.jsx

```diff
@@ -37,14 +37,13 @@ export default class Headers extends React.Component {
                 return null
               }
 
-              const description = header.get( "description" );
               const type = header.getIn(["schema"]) ? header.getIn(["schema", "type"]) : header.getIn(["type"])
               const schemaExample = header.getIn(["schema", "example"])
 
               return (<tr key={ key }>
                 <td className="header-col">{ key }</td>
                 <td className="header-col">{
-                  !description ? null : <Markdown source={ description } />
+                  !description ? null : <Markdown source={ header.get("description") } />
```

Comment:
Wouldn't the test of the unassigned `description` variable always resolve to `false` and as a result the description will never be displayed?

Reply:
@shockey I just saw that the Jenkins build was failing due to the removal of the `description` variable, so I put the variable back and now it passes the test suite.